packages/sdk/src/internal/messaging/data-service-messaging.ts

```diff
@@ -291,6 +291,11 @@ export class DataServiceThread implements Thread {
     }));
   }
 
+  async lastMessage(): Promise<ThreadMessage | null> {
```

Comment:
I think we still have n+1 problem here, probably we use some state var here instead of network call? Wdyt?

Reply:
Could you explain a little bit more about what the n+1 problem is? I'm a little fuzzy on that.